Costliest Natural Disasters Notebook Fix

mrbubbleman opened the pull request:
- fixed user-warning by using `.loc` for datetimes instead of `iloc`

byrcyb reviewed (commented):
The block that converts the DATE columns to date values ends up generating `NaT` values for the `EVENT START DATE` column, which causes the two graphs below it to show blank values.

mrbubbleman commented:
Error before was caused by not matching the string close enough, dropped all the dates instead of changing them to `datetime `objects. Changed the format to `'%m/%d/%Y %I:%M:%S %p'` to fix the errors.

misterhay reviewed (approved):
Looks like everything works, nice.
